docscan/sandbox/request/check/document_text_data_check_test.go

```diff
@@ -0,0 +1,72 @@
+package check
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/check/report"
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/filter"
+)
+
+func ExampleDocumentTextDataCheckBuilder() {
+	breakdown, err := report.NewBreakdownBuilder().
+		WithResult("some_result").
+		WithSubCheck("some_check").
+		Build()
+	if err != nil {
+		return
```

Comment:
Does returning here now make it quite hard to debug if there is an error? Would it be better to print the error? I imagine it will be displayed against the failing test
```suggestion
		fmt.Println(err.Error())
		return
```

Reply:
I think I copied approach in other examples in the SDK - no harm in adding the print if it helps 👍